app/javascript/app/components/ndcs/lts-explore-map/lts-explore-map-component.jsx

```diff
@@ -150,6 +150,7 @@ function LTSExploreMap(props) {
                         value={selectedCategory}
                         hideResetButton
                         plain
+                        showTooltip
```

Comment:
Maybe an improvement could be adding a condition for this to show if:
`showTooltip={selectedCategory && selectedCategory.length > 14}`
I think is quite annoying that it shows all the time. But I leave it up to you

Reply:
Nice suggestion, I pushed a commit with this tweak. Thanks! :+1: